assets/app/styles/_core.less

```diff
@@ -203,6 +203,19 @@ html, body {
   }
 }
 
+.nav-tabs {
+  font-size: @font-size-base;
+  a {
+    // ui.bootstrap.tabs doesn't set the href attr, so the cursor style is
```

Comment:
Try listening to $locationChangeStart or any of the other $location events
to fix the back button thing
On Oct 2, 2015 3:46 PM, "Sam Padgett" <notifications@github.com> wrote:

> In assets/app/styles/_core.less
> <https://github.com/openshift/origin/pull/4920#discussion_r41060340>:
>
> > @@ -203,6 +203,19 @@ html, body {
> >    }
> >  }
> >
> > +.nav-tabs {
> > +  font-size: @font-size-base;
> > +  a {
> > +    // ui.bootstrap.tabs doesn't set the href attr, so the cursor style is
>
> Hash URIs cause the router to reload, but it works well with query
> parameters if you set reloadOnSearch: false.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4920/files#r41060340>.
>